demos/nbody/main.cpp

```diff
@@ -66,17 +66,20 @@ class NBodyApp
     float max_radius = 25;
     float min_angle_pis = 0;
     float max_angle_pis = 2;
-    float min_height = -5;
-    float max_height = 5;
-    float lg_speed = 0;
+    float min_height = -50;
+    float max_height = 50;
+    float lg_speed = 0.4;
   } m_ui_distrib_cylinder_params;
 
   struct {
     float min_radius = 0;
     float max_radius = 25;
   } m_ui_distrib_sphere_params;
 
-  int32_t m_ui_n_bodies = 128;
+  int32_t m_ui_n_bodies = 8192;
+
+  const int32_t m_num_updates_per_frame = 1;
```

Comment:
I can guess what this variable does, but it's `const` and set to 1, is it to be expanded on?

Reply:
You can't change this value at moment using the UI so I added this as a way to group simulation steps together